sections/queueing_model/subsections/performance_measures/waiting_time/main.tex

```diff
@@ -0,0 +1,82 @@
+The set of accepting states for type 1 individuals is defined as:
+
+\begin{equation}\label{eq:accepting_states_type_1}
+    S_A^{(1)} = \{(u, v) \in S \; | \; v < N \}
+\end{equation}
+
+In essence, for type 1 individuals, this is the set of states that are not on 
+the last column of states in the Markov chain.
+Equivalently, the set of accepting states for type 2 individuals is defined as:
+
+\begin{equation}\label{eq:accepting_states_type_2}
+    S_A^{(2)}=
+    \begin{cases}
+        \{(u, v) \in S \; | \; u < M \}, & \textbf{if } T \leq N\\
+        \{(u, v) \in S \; | \; v < N \}, & \textbf{otherwise}
+    \end{cases}
+\end{equation}
+
+Note here that if the threshold is less than or equal the total capacity of the
+system the set includes all states that are not on the last column of the 
+Markov chain.
+Otherwise, the set of accepting state is identical to 
+(\ref{eq:accepting_states_type_1}). Thus, the expressions for the waiting times 
+for type 1 and type 2 individuals are given by:
+
+\begin{equation} \label{eq:closed_form_waiting_type_1}
+    W^{(1)} = \frac{\sum_{\substack{(u,v) \, \in S_A^{(1)} \\ v \geq C}} 
+    \frac{1}{C \mu} \times (v-C+1) \times \pi(u,v)}{\sum_{(u,v) \, 
+    \in S_A^{(1)}} \pi(u,v)}
+\end{equation}
+    
+\begin{equation}\label{eq:closed_form_waiting_type_2}
+    W^{(2)} = \frac{\sum_{\substack{(u,v) \, \in S_A^{(2)} \\ min(v,T) \geq C}} 
+    \frac{1}{C \mu} \times (\min(v+1,T)-C) \times \pi(u,v)}{\sum_{(u,v) \, 
+    \in S_A^{(2)}} \pi(u,v)}
+\end{equation}
+
+Consequently, the overall waiting time can be estimated by a linear combination 
+of \(W_1\) and \(W_2\). 
+Thus, the overall waiting time can calculated by the following equation where 
+\(c_1\) and \(c_2\) are the coefficients of the terms:
+
+\begin{equation}\label{eq:overall_waiting_time_coeff}
+    W = c_1 W^{(1)} + c_2 W^{(2)}
+\end{equation}
+
+The two coefficients represent the proportion of individuals of each type that 
+did not get lost and traversed through the model.
+Thus, one should account for the probability that an individual is lost to the 
+system. 
+This probability can be easily calculated by using the two sets of accepting 
+states \(S_A^{(2)}\) and \(S_A^{(1)}\) defined in equations 
+(\ref{eq:accepting_states_type_1}) and (\ref{eq:accepting_states_type_2}). 
+Using these equations the probability, for either individual type, that an 
+individual is not lost in the system is given by:
+
+\begin{equation*}
+    P(L'_1) = \sum_{(u,v) \, \in S_A^{(1)}} \pi(u,v) \hspace{2cm}
+    P(L'_2) = \sum_{(u,v) \, \in S_A^{(2)}} \pi(u,v)
+\end{equation*}
+ 
+Thus, by using these values as the coefficient of equation 
+(\ref{eq:overall_waiting_time_coeff}) the resultant equation can be used to get 
+the overall waiting time. 
+
+\begin{equation}\label{eq:overall_waiting_time}
+    W = \frac{\lambda_1 P(L'_1)}{\lambda_2 P(L'_2) + \lambda_1 P(L'_1)} W^{(1)} + 
+    \frac{\lambda_2 P(L'_2)}{\lambda_2 P(L'_2) + \lambda_1 P(L'_1)} W^{(2)}
```

Comment:
It took me a little while to realise that `L'_i` was not a parameter. Change the notation to be:

```
P_{L_1} 
```

and 

```
P_{L_2}
```

Reply:
I'm using `P(L'_i)` to define the probability that an individual of type `i` is NOT lost (so basically the probability of accepting an individual of type `i`). I did not use `P(A_i)` because I'm using the letter `A` for `S_A` (the set of accepting states).

I understand that what I have now is kind of ugly-looking but I think the solution would be to either explain better that `L'_i` is the negation of `L_i` or just use `P(A_i)`.

I don't fully understand why you said I should use `P_{L_i}`. I was literally talking about the: 
`Probability that a type i individual is not lost` which I believe translates to `P(L'_i)`

Reply:
`P(X)` makes it look like this is value parametrised by some parameter `X`. I'm fine with `P_{L'_i}`.

Reply:
Great 👍. Will use that then.